### PR TITLE
Bratseth/want to fail on operator fail

### PR DESCRIPTION
**Pull request by bratseth**

### CODE REVIEW

File: node-repository/src/main/java/com/yahoo/vespa/hosted/provision/NodeList.java [hunk 1]
```diff
@@ -277,6 +280,12 @@ private void ensureSingleCluster() {
     /** Returns the nodes of this as a stream */
     public Stream<Node> stream() { return asList().stream(); }
 
+    public static NodeList of(Node ... nodes) {
+        List<Node> nodeList = new ArrayList<>();
+        Collections.addAll(nodeList, nodes);
+        return copyOf(nodeList);
```
Comment (mpolden) [inline node-repository/src/main/java/com/yahoo/vespa/hosted/provision/NodeList.java:286]: Can be simplified to `return copyOf(List.of(nodes))`.

**Review by mpolden — APPROVED** [review]
LGTM, but build failed. I can't restart it without granting Screwdriver full access to my GH account.

File: node-repository/src/main/java/com/yahoo/vespa/hosted/provision/NodeList.java (hunk 1 above)
Comment (bratseth) [reply node-repository/src/main/java/com/yahoo/vespa/hosted/provision/NodeList.java]: Thanks, done

**Comment by bratseth** [comment]
Build error is unrelated - merging.

**Pull request merged**